global: adds rate limit section (#41)
zenodo/developers.zenodo.org: source/includes/rate-limit/_root.md

# Rate Limiting

> For our content and services to be available to everyone we have to make sure
> that our resources are being distributed fairly. 
> Due to the above we have implemented a rate limiter for our service which
> limits the requests that users can perform in a time window.
> Depending of the complexity of each request different endpoints can have
> different limits.

Reviewer: ```suggestion
For our content and services to be available to everyone we have to make sure that our resources are being distributed fairly. 

To achieve this, we have rate-limiting measures in place which limit the number of requests users can perform in a time window. Depending on the complexity and load of each request, different endpoints have different limits configured.
```

> Static pages in principle will have a very high limit.

Reviewer: I think we don't need to mention anything about static pages since most people are probably not aware of them.

Loading the thumbnail of a record is limited to 20 times per minute.

Harvesting the OAI-PMH endpoint is limited to 120 times per minute.

For the rest our policy allows 2000 requests per hour and 60 per minute for 
guest users.

Authenticated users have this limit increased to 5000 requests per hour and 

> 100 per minute.

Reviewer: I think these can be integrated into the table below (see example).

| Pages              | Limitations                                                                   |
|:------------------|:------------------------------------------------------------------------------|
| `static pages`   | No Limit. |

> | `OAI-PMH` | 120 requests per minute |

Reviewer: ```suggestion
| OAI-PMH API harvesting | 120 requests per minute |
```

> | `record thumbnails` | 20 requests per minute |

Reviewer: ```suggestion
| Thumbnails for image records | 20 requests per minute |
```

> | `global limit for guest user` | 60 requests per minute/ 2000 requests per hour |

Reviewer: I think backticks in this case are not needed, and we can use natural language.
